netlogo-gui/src/main/app/CloseButton.scala

```diff
@@ -0,0 +1,23 @@
+package org.nlogo.app
+
+import org.nlogo.swing.Utils
+import org.nlogo.theme.InterfaceColors
+
+import java.awt.{Dimension, Graphics}
+import javax.swing.JPanel
+
+private class CloseButton extends JPanel {
```

Comment:
Implement the `org.nlogo.swing.Transparent` trait, it's a helper I made to reduce the duplicate `setOpaque(false)` and `setBackground(InterfaceColors.TRANSPARENT)` calls everywhere.